app/models/response.rb

```diff
@@ -18,6 +18,10 @@ class Response < BaseModel
     end
   end
 
+  def show_secret
+    Digest::SHA256.hexdigest("#{uuid}#{Rails.config.secret_key_base}")
```

Comment:
Ik krijg hier een error: 
```
NoMethodError:
       undefined method `config' for Rails:Module
       Did you mean?  concerning
```
Ook in de console werkt `Rails.config` niet. Mis ik iets?